disco_aws_automation/disco_aws.py

```diff
@@ -79,7 +78,7 @@ def __init__(self, config, environment_name=None, boto2_conn=None, vpc=None, rem
         self._disco_storage = storage or None  # lazily initialized
         self._discogroup = discogroup or None  # lazily initialized
         self._autoscale = autoscale or None  # lazily initialized
-        self._elastigroup = elastigroup or None  # lazily initialized
+        # self._elastigroup = elastigroup or None  # lazily initialized
```

Comment:
This function will no longer work https://github.com/amplifylitco/asiaq/blob/3be9f67113619bc78274fdcf3cc89d10c1163dd8/disco_aws_automation/disco_aws.py#L152